style.css

```diff
@@ -84,54 +84,54 @@ html{font-family:sans-serif;-ms-text-size-adjust:100%;-webkit-text-size-adjust:1
 --------------------------------------------- */
 
 html,
-input[type="search"]{
+input[type="search"] {
 	-webkit-box-sizing: border-box;
-	-moz-box-sizing:    border-box;
-	box-sizing:         border-box;
+	-moz-box-sizing: border-box;
+	box-sizing: border-box;
 }
 
 *,
-*:before,
-*:after {
+*::before,
```

Comment:
You're right - there was already some `::before` in the file, so IE8 support looks to already have been dropped.